syntax = "proto3";

package planning;

import "common.proto";

message PaymentTask {
  string payment_task_id = 1;
  string user_id = 2;
  string transaction_id = 3;
  string account_id = 4;
  MetaData meta_data = 5;
}


message MetaData {
  PlanType preferred_plan_type =1;
  double preferred_timeline=2;
  PaymentFrequency preferred_payment_freq=3;
}